### ISSUE TITLE
Add api so external plugins can receive events when kubeconfig changes

### ISSUE THREAD

**Issue by lstocchi** [redhat-developer/intellij-kubernetes#165]
As in vscode, IJ Kubernetes should have an api so external plugins can receive events when the current context changes or the current namespace changes or the current kubeconfig path changes

**Comment by adietish** [comment]
@lstocchi: I guess that this became less in prio since intellij-kubernetes now change the kubeconfig (#193). Agree?

**Comment by lstocchi** [comment]
Yes, i guess we can also edit it or close it and reopen it in a different way. The only missing piece is if someone wants to use a different kubeconfig located somewhere else. The kubernetes extension should have a setting for it and then notify IJ-common, which should be able to handle this so the change is propagated to all other IJ extensions. WDYT?

**Comment by adietish** [comment]
We dont have external consumers of our plugin any more. Tekton- and Knative-Plugin were put into maintenance mode. Closing as OUTDATED.
